### DISCUSSION TITLE
Registering the WPF specific code without Splat

### DISCUSSION THREAD

**Discussion by adamradocz** [reactiveui/ReactiveUI#3318] [Q&A]
Hi,

Is there a way to register the WPF platform specific code without Splat?
I'm specifically interested in this: https://github.com/reactiveui/ReactiveUI/blob/c1b5ddcc54286ef9704da5d2941c03e08f06cd6b/src/ReactiveUI.Wpf/Registrations.cs#L18

**Comment by glennawatson** [comment]
No. Splat is pretty ingrained here. You can write your own splat adapter from the interface if you have more complex needs or need interior with existing code.
